htdocs/postdeploy.php

```diff
@@ -0,0 +1,68 @@
+<?php
```

Comment:
Alternatively, check if the project directory exists, and if so abort with an error right away..